### PR TITLE
Change config example that shows how to turn off process monitoring

### PR DESCRIPTION
**Pull request by dedemorton**
Cherry-pick proc monitoring example change into 1.1

### CODE REVIEW

File: topbeat/docs/gettingstarted.asciidoc [hunk 1]
```diff
@@ -107,13 +107,13 @@ By default, it's set to 10 seconds.
...
-If you are not interested in monitoring processes, you can use:
+If you are not interested in collecting per-process statistics, you can use:
 +
 [source, shell]
 -------------------------------------
 input:
-  period: 10
-  procs: ["^$"]
+  stats:
+    process: false
```
Comment (ruflin) [inline topbeat/docs/gettingstarted.asciidoc]: @monicasarbu is this `proc` or `process`? See https://github.com/elastic/beats/blob/1.1/topbeat/etc/topbeat.yml#L18

Comment (monicasarbu) [inline topbeat/docs/gettingstarted.asciidoc]: @ruflin Yes, it's process. I have updated it only in the tests. There was already a fix from the community and merged by @tsg: https://github.com/elastic/beats/pull/1343

**Comment by ruflin** [comment]
@dedemorton @monicasarbu This PR is against 1.1. The change from @tsg is against 1.2. So not sure if it is `process` in 1.1?

**Comment by dedemorton** [comment]
@ruflin I saw this commit and figured that the change was in for 1.1. Am I mistaken? https://github.com/elastic/beats/commit/a14ecfd799ac7d957bbffb5baabb110f3b3244fa

**Comment by ruflin** [comment]
@dedemorton Looks ok. But that would mean our config file for 1.1 is wrong: https://github.com/elastic/beats/blob/1.1/topbeat/etc/topbeat.yml#L18

**Comment by dedemorton** [comment]
@monicasarbu @ruflin I'm not sure how to proceed here. Is the config file wrong? If so, can we fix it, or do I need to document that it's wrong?

**Comment by ruflin** [comment]
I think the config in 1.1 is wrong and we should fix the config, not the docs, but I would like @monicasarbu to confirm.

**Comment by monicasarbu** [comment]
@ruflin yes, that's right. I think I fixed only the tests (to change from proc to process) and I didn't change the actual config in 1.1 :(

**Pull request merged**